homework-custom-lodash/omit.js

```diff
@@ -0,0 +1,13 @@
+const includes = require("./includes.js");
+
+const omit = function omitMethod(object, paths) {
+  const resObject = {};
+  for (let key in object) {
+    if (!paths.includes(key)) {
```

Comment:
`includes` is Array.prototype method 👮

Reply:
and also think about how to reuse `omit` in `omitBy` or vice versa

Reply:
oh, i wanted use my written includes but i wrote prototype method syntax =)